Beam/README.md

```diff
@@ -45,6 +45,7 @@ php artisan db:seed
 - PHP ^7.1.3
 - MySQL ^5.7
 - Redis ^3.2
+- Elastic ^6.8.1
```

Comment:
This is not completely correct. Here we list only dependencies for web admin (Laravel application) which is not dependent on Elastic directly. To state the dependency it would be better to state that `Segments API` is required (with the link to `#segments-go`).

The `Segments API` has the Elastic requirement already stated.

Reply:
ok, fixed